pyanaconda/bootloader/execution.py

```diff
@@ -30,20 +30,24 @@
 from pyanaconda.anaconda_loggers import get_module_logger
 log = get_module_logger(__name__)
 
-__all__ = ["BootloaderExecutor"]
+__all__ = ["setup_bootloader"]
```

Comment:
You have a typo in the commit message. 
``_setup_bootloaderset`` should be probably ``_setup_bootloader set``

Reply:
Fixed.